scripts/sync-and-test-bucket.sh

```diff
@@ -31,7 +31,17 @@ fi
 
 # For previews, name the destination bucket with the PR number, to reduce the number of
 # buckets we create and to facilitate shorter sync times.
-destination_bucket="$(origin_bucket_prefix)-$(build_identifier)"
+destination_bucket=""
+if [[ "$GITHUB_WORKFLOW" == "Build and deploy" ]]; then
+    echo "configuring bucket in production account"
+    destination_bucket="$(origin_bucket_prefix)-$(build_identifier)"
+elif [[ "$GITHUB_WORKFLOW" == "Build and deploy new account" ]]; then
+    echo "configuring bucket in new production account"
+    destination_bucket="www-$(origin_bucket_prefix)-$(build_identifier)"
```

Comment:
I'm pretty sure you're going to have to change this value to prevent bucket-naming collisions from happening as these workflows run in parallel (since they'll both be based on the same Git SHA): https://github.com/pulumi/docs/blob/a75eb1f1c6b87b0a36af90fb9bf7db622e024ab8/scripts/common.sh#L59